terraform/aws/analytical-platform-data-production/athena/iam-roles.tf

```
@@ -17,3 +17,9 @@ resource "aws_iam_role_policy_attachment" "athena_spark_s3_access" {
   role       = aws_iam_role.athena_spark_execution_role.name
   policy_arn = "arn:aws:iam::aws:policy/AmazonAthenaFullAccess"
 }
+
+resource "aws_iam_role_policy" "inline_policy" {
+  role   = aws_iam_role.athena_spark_execution_role.name
+  name   = "athena-spark-role-policy"
```

```suggestion
  name   = "athena-spark-dump-bucket-access"
```
Would be good to make the name less generic